BookingSpecBindings/Bindings/SignInPopUpBindings.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Threading;
+using BookingSpecBindings.TestBase.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support;
+using TechTalk.SpecFlow;
+
+namespace BookingSpecBindings.Bindings
+{
+	[Binding]
+	class SignInPopUpBindings
+	{
+		SignInPopUp signPage = new SignInPopUp();
+
+		[When(@"I set following parameters on Sign In Pop Up dialog")]
+		public void WhenISetFollowingParametersOnSignInPopUp(Table table)
+		{
+			foreach (var row in table.Rows)
+			{
+				string key = row["Field"];
+				switch (key)
+				{
+					case "Email":
+					signPage.TypeEmail(row["Value"]);
+						break;
+					case "Password":
+						signPage.TypePass(row["Value"]);
+						break;
+					default:
+						throw new NotImplementedException();
+				}
+			}
+		}
+		[When(@"I click Sign In button on PopUp")]
+		public void WhenIClickOnPopUpSignInButton()
+		{
+			signPage.ClickToSubmit();
+		}
+		[Then(@"I click on X button")]
+		public void ThenIClickOnXButton()
+		{
+			signPage.CloseSignInPopUp();
+		}
+		[Then(@"I see message error with ""(.*)""")]
+		public void ThenISeeMessageErrorWith(string message)
+		{
+			Assert.That(signPage.GetErrorText().Contains(message), Is.True);
```

Comment:
Contains() is not enough here. It's better to compare an entire message to catch possible issues with text.